public/scripts/controller/recipeController.js

```diff
@@ -34,5 +34,16 @@ var app = app || {};
     });
   };
 
+  recipeController.scrambleArray = (array) => {
```

Comment:
This seems like a private helper function, so I'd probably avoid attaching this to `recipeController` (meaning it's available outside this module).

Did you come up with this algorithm yourself?